tests/models.py

```diff
@@ -27,13 +28,20 @@ class Leak(models.Model):
     description = models.TextField()
 
 
+@python_2_unicode_compatible
 class SeaLion(SealableModel):
     height = models.PositiveIntegerField()
     weight = models.PositiveIntegerField()
     location = models.ForeignKey(Location, models.CASCADE, null=True, related_name='visitors')
     previous_locations = models.ManyToManyField(Location, related_name='previous_visitors')
     leak = models.ForeignKey(Leak, models.CASCADE, null=True)
 
+    def __str__(self):
+        return repr(self)
```

Comment:
`__str__` must now return text because of the `@python_2_unicode_compatible`, use `force_text(repr(self))`.